src/main/java/racing/core/Car.java

```diff
@@ -0,0 +1,10 @@
+package racing.core;
+
+public class Car {
+
+    private int position;
+
+    public Snapshot move(Move movement) {
+        return new Snapshot(movement.canMove() ? ++position : position);
```

Comment:
간단한 코드라서 메소드 분리할 필요는 없겠지만, 가독성을 위해서
```suggestion
        movement.canMove() ? ++position : position
        return new Snapshot(position);
```
로 표현하는 건 어떨까요?

Reply:
컴파일 오류를 방지하고자
```
        if (movement.canMove()) {
            ++position;
        }
        return new Snapshot(position);
```
로 변경했습니다!